gomint-server/src/main/java/io/gomint/server/inventory/item/ItemStainedGlassPane.java

```diff
@@ -3,18 +3,37 @@
 import io.gomint.inventory.item.ItemType;
 
 import io.gomint.server.registry.RegisterInfo;
-import io.gomint.taglib.NBTTagCompound;
+import io.gomint.world.block.Block;
+import io.gomint.world.block.BlockStainedGlassPane;
+import io.gomint.world.block.data.GlassColor;
 
 /**
  * @author geNAZt
  * @version 1.0
  */
+@RegisterInfo(sId = "minecraft:glass_pane")
 @RegisterInfo(sId = "minecraft:stained_glass_pane")
 public class ItemStainedGlassPane extends ItemStack< io.gomint.inventory.item.ItemStainedGlassPane> implements io.gomint.inventory.item.ItemStainedGlassPane {
 
     @Override
     public ItemType itemType() {
         return ItemType.STAINED_GLASS_PANE;
     }
+    @Override
+    public GlassColor color() {
+        return GlassColor.values()[this.data()];
+    }
 
+    @Override
+    public io.gomint.inventory.item.ItemStainedGlassPane color(GlassColor color) {
+        this.data((short) color.ordinal());
```

Comment:
Since the transparent has been merged to the end it would result in invalid data. Please handle that case different. Also you need to change the material here to glass_pane